feat: postgresql_anonymizer extension (#879)
aiven/aiven-docs: docs/products/postgresql/reference/list-of-extensions.md

    Show row-level locking information.
-   [pgstattuple](https://www.postgresql.org/docs/current/pgstattuple.html).
    Show tuple-level statistics.
-   [postgresql_anonymizer](https://postgresql-anonymizer.readthedocs.io/en/latest/).
    Mask or replace personally identifiable information (PII) or commercially sensitive
    data from a PostgreSQL database.

    :::note
    The `avnadmin` user who
    [installs the extension](/docs/products/postgresql/howto/manage-extensions#install-an-extension)

>     becomes its owner and is granted access to read internal tables and settings.

Reviewer: The wording here isn't entirely accurate. Any PostgreSQL user can install this extension, but we recommend installing it as `avnadmin`. The user who installs it becomes the extension owner.

Also, it's important to note:
1. Extension settings (e.g., `anon.salt`) can be modified by the `avnadmin` user (not necessarily the extension owner).
2. The settings can be read by the extension owner using the `anon.current_setting(TEXT)` function.

    Show row-level locking information.
-   [pgstattuple](https://www.postgresql.org/docs/current/pgstattuple.html).
    Show tuple-level statistics.
-   [postgresql_anonymizer](https://postgresql-anonymizer.readthedocs.io/en/latest/).
    Mask or replace personally identifiable information (PII) or commercially sensitive
    data from a PostgreSQL database.

    :::note

    The user who
    [installs the extension](/docs/products/postgresql/howto/manage-extensions#install-an-extension)
    (preferably `avnadmin`) becomes its **owner**.

>     - Updating extension settings (for example, `anon.salt`) is allowed for the
>       **avnadmin** user only, regardless of whether this avnadmin is the extension owner.

Reviewer 2: ```suggestion
      - Only the **avnadmin** user can update extension settings, such as `anon.salt`, 
        even if they are not the extension owner.  
```

    - Reading internal tables and settings is allowed for the extension **owner**,
      regardless of whether this owner is the avnadmin user. Use the

>       `anon.SETTING_NAME(TEXT)` function for that purpose.

Reviewer: Function is `anon.current_setting(TEXT)`. Here is an usage example: `anon.current_setting('anon.salt')`
